hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/SparkBroadcastManager.java

```diff
@@ -126,4 +146,24 @@ static Configuration getHadoopConfiguration(Configuration configuration) {
     }
     return (new HadoopStorageConfiguration(hadoopConf).getInline()).unwrap();
   }
+
+  static Map<String, String> getSchemaEvolutionConfigs(TableSchemaResolver schemaResolver,
+                                                       HoodieTimeline timeline,
+                                                       InstantFileNameGenerator fileNameGenerator,
+                                                       String basePath) {
+    Option<InternalSchema> internalSchemaOpt = schemaResolver.getTableInternalSchemaFromCommitMetadata();
+    Map<String, String> configs = new HashMap<>();
+    if (internalSchemaOpt.isPresent()) {
+      List<String> instantFiles = timeline.getInstants().stream().map(fileNameGenerator::getFileName).collect(Collectors.toList());
+      configs.put(SparkInternalSchemaConverter.HOODIE_VALID_COMMITS_LIST, String.join(",", instantFiles));
```

Comment:
Not sure why instant file name is needed, should we just just the instant timestamp string instead?

Reply:
@jonvex should be able to answer since Jon added this for fg reader.

Reply:
https://github.com/apache/hudi/blob/9da3221a79465f3326ae3ac206b08d60864ddcaa/hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieBaseRelation.scala#L678 I'm sure I just copied this method